Skyflow/src/main/kotlin/Skyflow/Padding.kt

```diff
@@ -1,3 +1,13 @@
 package Skyflow
 
-public class Padding(val left:Int=10,val top:Int=10,val right:Int=10,val bottom:Int=10) {}
\ No newline at end of file
+@Description("Sets the padding for the collect element which takes all the left, top, right, bottom padding values.")
```

Comment:
Padding for Collect Elements. Accepts left, top, right, and bottom padding values.